src/ts/ConvivaAnalytics.ts

```diff
@@ -319,6 +327,21 @@ export class ConvivaAnalytics {
     }
   }
 
+  private internalSetOverrides(metadataOverrides: Metadata) {
```

Comment:
Why the `internal` prefix?

Reply:
Actually just to match the naming scheme with `internalEndSession` / `internalInitializeSession` but it should then be `internalUpdateContentMetadata`.

I will rename it to `internalUpdateContentMetadata` to be consistent.